tests/unit/test-util/test.artifacts.js

```diff
@@ -55,4 +55,59 @@ describe('prepareArtifactsDir', () => {
     }
   ));
 
+  it('throws error when checking the directory if lacks writing permissions',
+    () => withTempDir(
+    (tmpDir) => {
+      const tmpPath = path.join(tmpDir.path(), 'build');
+      return fs.mkdir(tmpPath, '0622').then(() => {
```

Comment:
thanks for a clarifying (about arrow functions and in general). It is surprising that `win32` is true even for 64-bit systems